yojenkins/utility/utility.py

```diff
@@ -1033,3 +1034,22 @@ def get_item_action(item_info: dict, class_type: str) -> List[dict]:
                 actions_info.append(action)
 
     return actions_info
+
+
+def create_new_history_file(file_path: str) -> None:
+    """
+    Create a new blank command history file.
+
+    :param file_path: full path to the history file
+    :type file_path: str
+    """
+    try:
+        # Creating configuration directory if it does not exist
+        config_dir_abs_path = os.path.join(Path.home(), CONFIG_DIR_NAME)
+        if not os.path.exists(config_dir_abs_path):
+            print2("Configuration directory does not exist. Creating it ...")
+            os.makedirs(config_dir_abs_path)
```

Comment:
Hmmm... I wonder if we would need this then for `yojenkins auth configure`:
https://github.com/ismet55555/yojenkins/blob/main/yojenkins/yo_jenkins/auth.py#L236-L244

Since the history would take care of it. That `@log_to_history` decorator typically comes first before `yojenkins auth configure` can even happen, so the config directory `.yojenkins` would already be there
https://github.com/ismet55555/yojenkins/blob/main/yojenkins/cli/cli_auth.py#L16

Reply:
Yeah sounds like that bit of code is redundant. I have removed it from the PR in a specific commit, so you can always gitblame and back out that bit.